browser/extensions/api/brave_rewards_api.cc

```diff
@@ -1189,6 +1195,57 @@ BraveRewardsShouldShowOnboardingFunction::Run() {
   return RespondNow(OneArgument(base::Value(should_show)));
 }
 
+BraveRewardsGetScheduledCaptchaInfoFunction::
+    ~BraveRewardsGetScheduledCaptchaInfoFunction() = default;
+
+ExtensionFunction::ResponseAction
+BraveRewardsGetScheduledCaptchaInfoFunction::Run() {
+#if BUILDFLAG(BRAVE_ADAPTIVE_CAPTCHA_ENABLED)
+  Profile* profile = Profile::FromBrowserContext(browser_context());
+  auto* brave_adaptive_captcha_service =
+      brave_adaptive_captcha::BraveAdaptiveCaptchaServiceFactory::GetForProfile(
+          profile);
+  if (!brave_adaptive_captcha_service) {
+    return RespondNow(Error("Adaptive captcha service is not initialized"));
```

Comment:
This is not correct. If this is null it means you're calling this method from an incognito or other unsupported profile

Reply:
Fixed.